draft-ietf-rats-corim.md

```diff
@@ -151,17 +151,22 @@ This document specifies the information elements for representing Endorsements a
 
 # Introduction {#sec-intro}
 
-In order to conduct Evidence appraisal, a Verifier requires not only fresh Evidence from an Attester, but also trusted Endorsements (e.g., test results or certification data) and Reference Values (e.g., the version or digest of a firmware component) associated with the Attester.
-Endorsements and Reference Values are obtained from relevant supply chain actors, such as manufacturers, distributors, or device owners.
+RATS Architecture {{Section 4 of -rats-arch}}., specifies two important roles, Endorsers and Reference Value Providers.
+These roles are typically fulfilled by relevant supply chain actors, such as manufacturers, distributors, or device owners.
+They provide the required trusted Endorsements (e.g., test results or certification data) and Reference Values (e.g., the version or digest of a firmware component) about the Attester. This information is fundamental for correct operation of a Verifier (i.e. appraising Evidence received from the Attester).
+
 In a complex supply chain, multiple actors will likely produce these values over several points in time.
-As such, one supply chain actor will only provide the subset of characteristics that they know about the Attester. A proper subset is typical because a certain supply chain actor will be the responsible authority for only a system component/module that is measured amongst a long chain of measurements.
+As such, one supply chain actor will only provide the subset of characteristics that they know about the Attester. A proper subset is typical because a certain supply chain actor will be the responsible authority for only a system component/module
```

Comment:
```suggestion
As such, one supply chain actor might only supply a portion of the Reference Values or Endorsements that otherwise fully characterizes an Attester.
Ideally, only the supply chain actor who is the most knowledgable entity regarding a particular component will supply Reference Values or Endorsements for that component.
```